FairSoft_test.cmake

```diff
@@ -123,31 +133,41 @@ if (NOT "$ENV{BRANCH_NAME}" STREQUAL "")
 endif()
 message(STATUS " cdash_group ..........: ${cdash_group}")
 
-
-# Full cleaning:
-#  ctest_empty_binary_directory("${CTEST_BINARY_DIRECTORY}")
-#
-# The cache contains the name of the source directory.
-# As we might create a temporary source directory, this does
-# not any more match. So remove the cache before start / configure.
-file(REMOVE "${CTEST_BINARY_DIRECTORY}/CMakeCache.txt")
-# Remove workdir (keep in sync with CMakeLists.txt)
-file(REMOVE_RECURSE "${CTEST_BINARY_DIRECTORY}/test_workdir")
-
-
-ctest_start(Continuous TRACK ${cdash_group})
-ctest_configure(OPTIONS "-DFS_TEST_WORKDIR=${FS_TEST_WORKDIR};-DFS_TEST_INSTALLTREE=${FS_TEST_INSTALLTREE}")
-# ctest_submit(PARTS Start Configure)
-
-# ctest_build()
+if (BUILD_METHOD STREQUAL legacy)
+  ctest_empty_binary_directory(${CTEST_BINARY_DIRECTORY})
```

Comment:
on macOS, the (to be cached) install-tree is in `build/install-tree`. ;)
So this might kill the install-tree.

I think, I should write up a "test architecture" docs section. ;-)

Reply:
*grrr* I missed the `if (BUILD_METHOD STREQUAL legacy)` above it.

Reply:
ya, i suspected something like this. any arguments against moving `install-tree` besides the build dir?